packages-content-model/roosterjs-content-model-editor/lib/editor/plugins/ContentModelEditPlugin.ts

```diff
@@ -91,7 +91,11 @@ export default class ContentModelEditPlugin implements EditorPlugin {
                     break;
 
                 case PluginEventType.KeyDown:
-                    this.handleKeyDownEvent(this.editor, event.rawEvent);
+                    this.handleKeyDownEvent(
+                        this.editor,
+                        event.rawEvent,
```

Comment:
Let's pass the whole event object in, so no need to pass in two parameter separately.